charts/cortx/README.md

```diff
@@ -107,7 +105,7 @@ helm uninstall cortx
 | cortxcontrol.machineid.value | string | `""` |  |
 | cortxcontrol.service.loadbal.enabled | bool | `true` |  |
 | cortxcontrol.service.loadbal.nodePorts.https | string | `""` |  |
-| cortxcontrol.service.loadbal.ports.https | int | `8081` |  |
+| cortxcontrol.service.loadbal.ports.https | int | `23256` |  |
```

Comment:
I don't think this change should be made here, as this would change the port on `cortx-control-loadbal-svc` Service to listen on 23256 and I believe it is still expected to be on 8081 by default. It can forward to 23256 in the container, which you are also doing, but this would change the port the service itself is listening on - which I don't think is desired.

Reply:
To conclude, nothing in the cluster is currently using port 8081, or at least not explicitly. We don't actually write this section to the config.yaml yet, and its purpose isn't clear to me:

```yaml
csm:
    public:                                                    # Public Control Endpoint
      endpoints:
      - https://control-svc:8081
```

But, based on that [sample](https://github.com/Seagate/cortx-prvsnr/blob/325a3e0bb9ef36de6d918d9167ecdabaf4f681d2/conf/solution/config.yaml.sample#L154-L156), we should continue to configure the service port in that way.